app/helpers/tab_helper.rb

```diff
@@ -73,10 +73,10 @@ def nav_link(options = {}, &block)
   end
 
   # Use nav_tab for save controller/action  but different params
-  def nav_tab key, value, &block
+  def nav_tab(*keys, value, &block)
     o = {}
     o[:class] = ""
-    o[:class] << " active" if params[key] == value
+    o[:class] << " active" if keys.detect { |k| params[k] == value }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.